nurseconnect/templatetags/nurseconnect_tags.py

```diff
@@ -0,0 +1,22 @@
+from django.template import Library
+
+from molo.core.templatetags.core_tags import get_pages
+
+register = Library()
+
+
+@register.inclusion_tag('core/tags/footerlink.html', takes_context=True)
```

Comment:
The tag `footerpage` renders each footer that exists on the CMS. Thus, in order to only access the terms and condition link, I had to create another template tag called `footerlink`. Since the CMS pages cannot be accessed through a regular template such as `register.html`